invenio_rdm_records/services/tasks.py

```diff
@@ -27,3 +45,56 @@ def update_expired_embargos():
         except EmbargoNotLiftedError as ex:
             current_app.logger.warning(ex.description)
             continue
+
+
+@shared_task(ignore_result=True)
+def reindex_stats(stats_indices, max_bucket_size=10000):
+    """Reindex the documents where the stats have changed."""
+    bm = BookmarkAPI(current_search_client, "stats_reindex", "day")
+    last_run = bm.get_bookmark()
```

Comment:
I am a bit confused: every time the aggregations are computed, the bookmarks is updated.
IMO, the last run should correspond to the last run of this task, using a Redis cache unique key to register that.
Let's discuss this IRL.